spec/models/publication_spec.rb

```diff
@@ -346,19 +346,25 @@
   end
```

Comment:
This commit and the previous commit "Finer test targeting" are dependent commits, might be good to squash them.